triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -497,6 +519,21 @@ object Runner extends StrictLogging {
     }
   }
 
+  private[this] val DaTypesTuple2 =
```

Comment:
Maybe a better fit for the converter? Probably even the shared converter.

Reply:
f8285a579128d7151ff30a661b5241dc4b87a9b8